pages/gigabit/Project.jsx

```diff
@@ -52,12 +60,14 @@ module.exports = React.createClass({
           </div>
         </div>
 
-        <div className="m-b-3 text-center">
-          <p>Learn more about {data.Project}</p>
-          <div className="middle-button">
-            <a className="secondary-button" href={data.Link} target="_blank">Website</a>
+        { data.Links &&
```

Comment:
Looks good but I would change this from `data.Links` to `links` since we already checked and initialized `links` based on `data.Links` on line 16.